src/modules/app/app.css

```diff
@@ -0,0 +1,38 @@
+@import '../core/core.css';
+@import '../icon/icon.css';
+@import '../info-panel/info-panel.css';
+@import '../module-selector/module-selector.css';
+@import '../size-controls/size-controls.css';
+
+app {
+    font-family: Helvetica, Arial, sans-serif;
+    line-height: 2.5;
+}
+
+.app-header {
+    padding: 0 0.5em;
+    background-color: var(--color-grey);
+    border-bottom: 1px solid var(--color-grey-dark);
+    box-shadow: 0 0 5px rgba(0, 0, 0, 0.2);
+}
+
+.app-header > *:not(:last-child) {
```

Comment:
`.app-header > * + * { margin-left: ... }` does the same thing but is more expressive I think. As what it says is: "add some space before the next element". That's what a designer would say. Not "add some space after each element except for the last element". Also `* + *` has better browser support, but doesn't matter for IE9+ scope.